### PR TITLE
WIP implement po editor

### PR DESCRIPTION
**Pull request by vharitonsky**

### CODE REVIEW

File: my-app/README.md [hunk 1]
```diff
@@ -0,0 +1,2164 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```
Comment (AlexMost) [inline my-app/README.md]: I think we don't need README for create react app here)

File: my-app/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  }
+
+  iterMessages = function* (m: Messages): IterableIterator<Message> {
+    for(const key of Object.keys(m)){
+      yield m[key];
+    }
+  }
+
+  render() {
+    console.log(this.state.poData)
```
Comment (AlexMost) [inline my-app/src/App.tsx]: log detectd

File: my-app/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  constructor(props:any) {
+    super(props)
+  }
+
+
+  onDrop(evt: DragEvent<HTMLDivElement>) {
+    evt.stopPropagation();
+    evt.preventDefault();
+
+    var files = evt.dataTransfer.files; // FileList object.
```
Comment (AlexMost) [inline my-app/src/App.tsx]: let ?

File: my-app/src/App.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+    // files is a FileList of File objects. List some properties.
+    for (var i = 0, f; f = files[i]; i++) {
```
Comment (AlexMost) [inline my-app/src/App.tsx]: let ?

File: my-app/src/App.tsx (hunk 3 above)
Comment (vharitonsky) [reply my-app/src/App.tsx]: lazy copy-paste

File: my-app/src/App.tsx (hunk 4 above)
Comment (vharitonsky) [reply my-app/src/App.tsx]: lazy copy-paste

**Pull request merged**